src/main/java/com/github/princesslana/eriscasper/data/Data.java

```diff
@@ -35,4 +38,16 @@ public static String toJsonString(Object obj) throws DataException {
       throw new DataException(e);
     }
   }
+
+  public static String toQueryString(Object obj) {
+    JsonNode node = jackson.valueToTree(obj);
+    Map<String, String> queryMapper = new HashMap<>();
+    node.fields()
+        .forEachRemaining(entry -> queryMapper.put(entry.getKey(), entry.getValue().asText()));
```

Comment:
What's the difference between doing this, and doing `fromJson(jackson.valueToTree(obj), Map.class)` ?

Reply:
It seems like it would be a lot heavier.

Reply:
You would be able to have jackson deserialize into a `Map<String, Object>`, and then use `toString` on the values.

Reply:
See the latest commit. I used that principle.